tests/fixtures/scenarios/exitCode/components/terraform/empty/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "my_param" {
+  description = "A test parameter for the null resource"
+  type        = string
+  default     = "default_value"
+}
+variable "stage" {
```

Comment:
had to add this because of above issue

Reply:
The `name_pattern` defines how the stack IDs are generated.

```
 name_pattern: "{stage}"
```

If you remove stage, then it doesn't work. 

Instead, change this to `name_template`, and look at some of the other examples that use the `mock` component instead.

Reply:
thanks for this. really helped me...